src/EPR.RegulatorService.Facade.Core/Services/Regulator/RegulatorOrganisationService.cs

```diff
@@ -106,19 +110,19 @@ public async Task<HttpResponseMessage> RegulatorEnrollment(EnrolInvitedUserReque
 
         public async Task<HttpResponseMessage> RegulatorInvited(Guid id, string email)
         {
-            _logger.LogInformation("Attempting to fetch pending invite token from the backend");
+            _logger.LogInformation("Attempting to fetch pending applications from the backend");
 
             var url = string.Format($"{_config.Endpoints.RegulatorInvitedUser}", id, email);
 
-            return await _httpClient.GetAsync(url);
+            return await _httpClient.GetAsync(CheckURL(url));
```

Comment:
why not use that other pattern?
var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
{
Path = url
};

Reply:
The url is escaped and does not format correctly. I did not want to change the test.

Reply:
Please undo this change, ignore the SQ warning as peter is happy to leave it as is